README.md

```diff
@@ -1,47 +1,43 @@
-[nuget-url]: https://nuget.org/packages/McMaster.AspNetCore.LetsEncrypt
+[nuget-badge]: https://img.shields.io/nuget/v/LettuceEncrypt?color=blue
+[nuget-url]: https://nuget.org/packages/LettuceEncrypt
+[ACME]: https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment
+[Let's Encrypt]: https://letsencrypt.org/
 
-[Let's Encrypt](https://letsencrypt.org/) is a free, automated, and open Certificate Authority.
-This project provides API for ASP.NET Core projects to use Let's Encrypt.
+LettuceEncrypt 🥬 provides API for ASP.NET Core projects to integrate with a certificate authority (CA), such as
+[Let's Encrypt], for free, automatic HTTPS (SSL/TLS) certificate generation using the [ACME] protocol.
 
-When enabled, your web server will use the Let's Encrypt certificate authority
-and **automatically** generate an HTTPS certificate when the server starts up. It then configures Kestrel to use this certificate for all HTTPs traffic. See [usage instructions below](#usage) to get started.
+When enabled, your web server will **automatically** generate an HTTPS certificate during start up. It then configures Kestrel to use this certificate for all HTTPS traffic. See [usage instructions below](#usage) to get started.
 
-Created and developed by [@natemcmaster](https://github.com/natemcmaster) with :heart: from Seattle :coffee:
+Created and developed by [@natemcmaster](https://github.com/natemcmaster) with ❤️from Seattle ☕️.
+This project was formerly known as "LetsEncrypt", but has been renamed for
+trademark reasons. This project is **not an official
+offering** from Let's Encrypt® or ISRG™.
 
 Special thanks to my sponsors!
 
 * [@bordenit](https://github.com/bordenit)
 
+This project is 100% organic and best served cold with ranch and carrots.
+
 ## Will this work for me?
 
 That depends on [which kind of web server you are using](#web-server-scenarios). This library only works with [Kestrel](https://docs.microsoft.com/aspnet/core/fundamentals/servers/kestrel), which is the default server configuration for ASP.NET Core projects. Other servers, such as IIS and HTTP.sys, are not supported. Furthermore, this only works when Kestrel is the edge server.
 
 Not sure? [Read "Web Server Scenarios" below for more details.](#web-server-scenarios)
 
-Using :cloud: Azure App Services (aka WebApps)? This library isn't for you, but you can still get free HTTPS certificates from Let's Encrypt. See ["Securing An Azure App Service with Let's Encrypt"](https://www.hanselman.com/blog/SecuringAnAzureAppServiceWebsiteUnderSSLInMinutesWithLetsEncrypt.aspx) by Scott Hanselman for more details.
+Using :cloud: Azure App Services (aka WebApps)? This library isn't for you, but you can still get free HTTPS certificates. See ["Securing An Azure App Service with Let's Encrypt"](https://www.hanselman.com/blog/SecuringAnAzureAppServiceWebsiteUnderSSLInMinutesWithLettuceEncrypt.aspx) by Scott Hanselman for more details.
```

Comment:
I don't think you wanted to change that link

Reply:
Good catch!